test/extended/kubevirt/nodereboot.go

```diff
@@ -0,0 +1,85 @@
+package kubevirt
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	k8sapierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = Describe("[sig-kubevirt] node reboot", func() {
+	oc := exutil.NewCLIWithPodSecurityLevel("ns-global", admissionapi.LevelBaseline)
+
+	InKubeVirtClusterContext(oc, func() {
+		mgmtFramework := e2e.NewDefaultFramework("mgmt-framework")
+		mgmtFramework.SkipNamespaceCreation = true
+
+		hostedFramework := e2e.NewDefaultFramework("hosted-framework")
+		hostedFramework.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
+
+		When("an hosted control plane worker is rebooted", func() {
```

Comment:
```suggestion
		When("a hosted control plane worker is rebooted", func() {
```

Reply:
I think the h in hosted in silent, thus this should be `an`. 

... but I'm not a native speaker . 

I'll defer to one.